build.gradle

```diff
@@ -40,7 +40,8 @@ ext {
   // https://github.com/palantir/gradle-consistent-versions/issues/383
   scriptDepVersions = [
       "apache-rat": "0.11",
-      "jflex": "1.7.0"
+      "jflex": "1.7.0",
+      "javacc": "5.0"
```

Comment:
OK, I'm not quite sure why I can't reply to your comment about the misleading message so I'll combine them.

Yeah, **File "TokenMgrError.java" does not exist.  Will create one.** is because the build file deletes the outputs first. If they _aren't_ deleted, you get output about "file is obsolete, please delete or rename it". So I have no faith that javacc really overwrites them.

Besides, if for some mysterious reason the generated files ever change, this'll delete the no-longer-generated ones. Unlikely I admit...

Dawid alphabetized this already.